src/emoji.ts

```diff
@@ -15,14 +15,14 @@ limitations under the License.
 */
 
 import EMOJIBASE from 'emojibase-data/en/compact.json';
+import SHORTCODES from 'emojibase-data/en/shortcodes/iamcal.json';
 
 export interface IEmoji {
     annotation: string;
-    group: number;
+    group?: number;
     hexcode: string;
-    order: number;
-    shortcodes: string[];
-    tags: string[];
+    order?: number;
+    tags?: string[];
     unicode: string;
     emoticon?: string;
 }
```

Comment:
I think we should zip the shortcodes in here, as we pretty much always pass around references to IEmoji so it'll act as a cheaper pointer than a map/object lookup